src/applications/edu-benefits/1995/helpers.jsx

```diff
@@ -27,3 +28,63 @@ export const display1995StemFlow = form =>
   isEdithNourseRogersScholarship(form) &&
   (isEligibleForEdithNourseRogersScholarship(form) ||
     determineEligibilityFor1995Stem(form));
+
+export const buildSubmitEventData = formData => {
+  if (environment.isProduction()) {
+    const exhaustedAllBenefits =
+      formData['view:exhaustionOfBenefits'] === true ||
+      formData['view:exhaustionOfBenefitsAfterPursuingTeachingCert'] === true;
+    return {
+      benefitsUsedRecently: formData.benefit,
+      'edu-stemApplicant': formData.isEdithNourseRogersScholarship
+        ? 'Yes'
+        : 'No',
+      'edu-undergradStem': formData.isEnrolledStem ? 'Yes' : 'No',
+      'edu-pursueTeaching': formData.isPursuingTeachingCert ? 'Yes' : 'No',
+      activeDuty: formData.isActiveDuty ? 'Yes' : 'No',
+      calledActiveDuty: formData.isActiveDuty ? 'Yes' : 'No',
+      preferredContactMethod: formData.preferredContactMethod,
+      'edu-exhaustedAllBenefits': exhaustedAllBenefits ? 'Yes' : 'No',
+    };
+  }
+
+  const yesNoOrUndefined = value => {
+    if (value === undefined) {
+      return undefined;
+    }
+    return value ? 'Yes' : 'No';
+  };
+
+  return {
+    'benefits-used-recently': formData.benefit,
+    'new-service-periods-to-record': yesNoOrUndefined(
+      formData['view:newService'],
+    ),
+    'service-details': _.get(formData, 'toursOfDuty', []).map(tour => ({
```

Comment:
You don't need to import lodash for this.  Try this instead:

```js
     'service-details':  (formData.toursOfDuty || []).map(tour => ({
```

Reply:
Thanks!